doc/visual-programming/source/widgets/data/featureconstructor.md

```diff
@@ -11,7 +11,9 @@ Add new features to your dataset.
 
 - Data: dataset with additional features
 
-The **Feature Constructor** allows you to manually add features (columns) into your dataset. The new feature can be a computation of an existing one or a combination of several (addition, subtraction, etc.). You can choose what type of feature it will be (discrete, continuous or string) and what its parameters are (name, value, expression). For continuous variables you only have to construct an expression in Python.
+**Formula** allows computing new columns by combining the existing ones with a user-defined expression. The resulting column can be categorical, numerical or textual.
+
+For numeric variables, it sufices to provide a name and an expression.
 
 ![](images/feature-constructor1-stamped.png)
```

Comment:
feature-construtor2 was updated. Should this one also be updated?

Reply:
It's unstamped. I guess not.

Reply:
I changed feature-constructor2 because of changes in the widget's interface, which occurred before this PR. In feature-constructor1, just the window title is different, so I think it's not necessary. (I hesitate changing screenshots to often to avoid needlessly increasing the repo size.)